test/integration/data/intersection/restored.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Did you accidentally commit these files, or are they used in a test that I can't find?

Reply:
Oh nevermind, I found the tests.

Reply:
Oh, yeah; there's a whole data-driven test thing going on in this directory